biggraphite/drivers/elasticsearch.py

```diff
@@ -434,7 +439,9 @@ def insert_downsampled_points_async(self, metric, datapoints, on_done=None):
     def drop_all_metrics(self, *args, **kwargs):
         """See the real Accessor for a description."""
         super(_ElasticSearchAccessor, self).drop_all_metrics(*args, **kwargs)
-        pass
+        # Drop indices.
+        self.client.indices.delete('*')
```

Comment:
use the index prefix here, you don't want to delete *everything*

Reply:
Done